packages/app-cli/app/command-mv.js

```diff
@@ -7,25 +7,38 @@ const Note = require('@joplin/lib/models/Note').default;
 
 class Command extends BaseCommand {
 	usage() {
-		return 'mv <note> [notebook]';
+		return 'mv <note_or_notebook> [notebook]';
 	}
 
 	description() {
-		return _('Moves the notes matching <note> to [notebook].');
+		return _('Moves the notes matching <note_or_notebook> or single notebook to [notebook].');
 	}
 
 	async action(args) {
-		const pattern = args['note'];
+		const pattern = args['note_or_notebook'];
 		const destination = args['notebook'];
 
-		const folder = await Folder.loadByField('title', destination);
-		if (!folder) throw new Error(_('Cannot find "%s".', destination));
-
-		const notes = await app().loadItems(BaseModel.TYPE_NOTE, pattern);
-		if (!notes.length) throw new Error(_('Cannot find "%s".', pattern));
+		// TODO: allow partial ids
+		let folder = await Folder.loadByField('id', destination);
+		if (!folder) {
+			folder = await Folder.loadByField('title', destination);
+			if (!folder) throw new Error(_('Cannot find "%s".', destination));
+		}
```

Comment:
Aren't partial IDs already handled by loadItem/loadItems in app.js? The problem is that you use Folder.loadByField and thus bypass all the business logic implemented in the app().loadItems call you removed.